lib/runners/recommended_config.rb

```diff
@@ -0,0 +1,26 @@
+module Runners
+  module RecommendedConfig
+    def warn_recommended_config_file_release(config_filename, release_date)
+      add_warning <<~MSG, file: config_filename
+        Sider's recommended configuration file is about to release in #{release_date}.
```

Comment:
I think _release_ usually requires an _object_, and it seems that its _object_ would be "Sider's recommended configuration file." So, using passive voice looks better.

```suggestion
        Sider's recommended configuration file is about to be released in #{release_date}.
```

Reply:
Fixed. commit 0b6f185ad3f767c32f49b988fa10629fb2837ce4